### ISSUE TITLE
Why is it possible to do multiple classifications

### ISSUE THREAD

**Issue by QiuSYang** [reachsumit/deep-unet-for-satellite-image-segmentation#11]
I find the gt_mband datasets only have one label, Why is it possible to do multiple classifications?
in addition, How do you do multiple classification using the binary crossentropy loss function

**Comment by prativadas** [comment]
This is a multi label classification problem not multi class classification,in multi label classification problem you can have a single picture that has more than 1 target label unlike multi class classification problem in which every entry has only 1 label. Multi-label classification you need to use binary cross entropy.
